### ISSUE TITLE
East Asian ambiguous character width

### ISSUE THREAD

**Issue by GoogleCodeExporter** [mintty/mintty#88]
```
This patch adds "CJK width" option that supports "East Asian Ambiguous
Character Width". ( http://unicode.org/reports/tr11/ )

When this option is checked, the width of some Unicode characters is 2.

This patch:
1. Add new option "CJK width" to MinTTY Options dialog.
2. Markus Kuhn's wcwidth.c separated from unicode.c.
3. wcwidth.c upgraded to the 2007-05-26 (Unicode 5.0) edition.
4. Some patches are applied to wcwidth.c.

```

Original issue reported on code.google.com by `deenhe...@gmail.com` on 15 Apr 2009 at 3:01

Attachments:
- [cjkwidth.patch](https://storage.googleapis.com/google-code-attachments/mintty/issue-88/comment-0/cjkwidth.patch)

**Comment by mintty** [comment]
Released 2.8.1 with new option `Charwidth=ambig-wide` to activate built-in ambiguous-wide handling.

**Comment by mintty** [comment]
Released 3.1.5 with explicit character width attribute support.
